modules/chapter2/pages/section1.adoc

```diff
@@ -1,3 +1,76 @@
-= Section 1
+= Data Science Projects
 
-This is _Section 1_ of _Chapter 2_ in the *hello* quick course....
\ No newline at end of file
+In RHODS, a _data science project_ is the preferred way to organize resources when working on an AI/ML application.
+Internally, a data science project is an OpenShift project that contains specific labels to identify the project as a RHODS-specific project,
+for example:
+
+[source,yaml,subs="+quotes"]
+----
+kind: Project
+apiVersion: project.openshift.io/v1
+metadata:
+  name: my-ai-project
+  labels:
+    kubernetes.io/metadata.name: my-ai-project
+    *modelmesh-enabled: 'true'*
+    *opendatahub.io/dashboard: 'true'*
+    ...
+----
+
+== The Need for Data Science projects
+
+Similarly to how you use projects in OpenShift for other workloads, you can use and should use data science projects to organize the different elements that you need for your AI applications, such as workbenches, model servers, or persistent storage.
+
+You can create and manage data science projects by using the RHODS dashboard.
+
+== Exercise
+
+=== Creating Data Science Projects
+
+First, open the RHODS dashboard.
+You might need to enter your credentials to access the dashboard.
+
+To create a data science project, click btn:[Data Science Projects] in the left navigation pane.
+
+image::rhods-side-menu.png[width=50%]
+
+In the btn:[Data science projects] page, click btn:[Create data science project].
+
+In the modal window that opens, enter the name of the project and click btn:[Create].
+You can optionally add a description and modify the OpenShift resource name for the project.
+
+image::create-ds-project.png[]
+
+After you create the project, RHODS opens the project dashboard page.
+
+image::ds-project-dashboard.png[]
+
+From this page you can manage the project components and the permissions.
+
+
+=== Configure Access
```

Comment:
Add a TODO marker - Link to the admin course section that teaches user and groups creation